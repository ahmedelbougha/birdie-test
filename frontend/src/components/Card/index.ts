export { Card as default } from "./Card";
export { type CardProps } from "./Card.d";
export {
    StyledCard,
    StyledCardDivider,
    StyledCardsWrapper as CardsWrapper,
    StyledDashboardCard
} from "./Card.styled";
export { default as DashboardCard } from "./DashboardCard";

