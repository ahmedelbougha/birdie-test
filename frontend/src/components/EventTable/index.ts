export { EventTable as default } from "./EventTable";
export { type TableProps } from "./EventTable.d";
export { StyledTable } from "./Table.styled";

