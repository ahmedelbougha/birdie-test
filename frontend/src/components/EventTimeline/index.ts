export { EventTimeline as default } from "./EventTimeline";
export { type EventTimelineElementProps, type EventTimelineProps } from "./EventTimeline.d";
export { default as EventTimelineElement } from "./EventTimelineElement";

