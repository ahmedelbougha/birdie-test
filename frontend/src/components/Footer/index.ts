export { Footer as default } from "./Footer";
export {
    StyledFooter,
    StyledFooterLogo
} from "./Footer.styled";

