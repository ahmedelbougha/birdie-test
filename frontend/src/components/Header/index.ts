export { Header as default } from "./Header";
export {
    StyledHeader,
    StyledHeaderLogo,
    StyledNav
} from "./Header.styled";

