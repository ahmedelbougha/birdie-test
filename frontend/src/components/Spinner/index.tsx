import { Container } from '../styles/Container.styled';
import StyledSpinner from './Spinner.styled';

export default function Spinner() {
  return (
    <Container>
      <StyledSpinner />
    </Container>
  );
}
