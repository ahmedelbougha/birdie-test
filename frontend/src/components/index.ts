export { CardsWrapper, DashboardCard, default as Card } from "./Card";
export { default as EventTable } from "./EventTable";
export { default as EventTimeline, EventTimelineElement } from "./EventTimeline";
export { default as Footer } from "./Footer";
export { default as Header } from "./Header";
export { default as Spinner } from "./Spinner";
export { Button, Container, GlobalStyles, Layout, StyledError } from "./styles";

