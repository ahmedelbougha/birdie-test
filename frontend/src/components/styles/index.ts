export { Button } from "./Button.styled";
export { Container } from "./Container.styled";
export { StyledError } from "./Error.styled";
export { GlobalStyles } from "./Global.styled";
export { Layout } from "./Layout.styled";

