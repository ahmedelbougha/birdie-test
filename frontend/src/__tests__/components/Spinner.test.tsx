import renderer from 'react-test-renderer';
import { Spinner } from '../../components';

describe('Spinner', () => {
  it('renders', () => {
    const component = renderer.create(<Spinner />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
