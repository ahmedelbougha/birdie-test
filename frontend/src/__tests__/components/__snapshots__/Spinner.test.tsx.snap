// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner renders 1`] = `
<div
  className="sc-bcXHqe iuWKlD"
>
  <div
    className="sc-gswNZR jcHCHp"
  />
</div>
`;
